Sources/Testing/Running/EntryPoint.swift

```diff
@@ -260,7 +260,9 @@ extension [Event.ConsoleOutputRecorder.Option] {
 #endif
 
     // Load tag colors from user/package preferences on disk.
-    result += tagColorOptions()
+    if let tagColors = try? loadTagColors() {
```

Comment:
If there's an error thrown here should we log at least, while proceeding to run tests? (I see this is not really a behavior change from before this PR, but might be nice, still)

Reply:
_Maybe_. I don't want to leak a dependency on Foundation out of that file, but we'd need to check for `CocoaError(.fileNotFound)` (or however it's spelt.)

Reply:
Oh I figured it could just handle _any_ error, not necessarily specific ones. Tag colors aren't mission critical, I wouldn't say, so any error that arises could just be noted but we continue to run tests

Reply:
That wouldn't work because it throws an error if the user hasn't created a "tag-colors.json" file, which is obviously not a failure mode. Hence we'd need to special-case file-not-found errors.

Reply:
Maybe separating out the "is there a colors file" and "parse said file" logic would make that easier? Seems like the 
former could be non-throwing

Reply:
That would require us to leak the Foundation dependency out in the form of returning an instance of `Data?` which, again, we want to avoid.

Since, as you said, tag colors aren't mission critical, we're probably overthinking it. :)